### PR TITLE
Update README file for testnet

### PR DESCRIPTION
**Pull request by jsdanielh**
Update the `README.md` file for the testnet release.

## Pull request checklist

- [X] All tests pass. The project builds and runs.
- [X] I have resolved any merge conflicts.
- [X] I have resolved all `clippy` and `rustfmt` warnings.

### CODE REVIEW

File: docs/becoming_validator.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+Once in the session, check the status with:
+
+```
+status
+```
+
+## 3. Become a Validator
+
+To become a validator, the process is to send a _crate_ validator transaction. For that you need to have an account
```
Comment (Eligioo) [inline docs/becoming_validator.md]: Typo: crate --> create

File: docs/becoming_validator.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+```
+
+Note that this `validator_private_key` is the private key of the Schnorr keypair for the validator address.
+
+### 3.2 Unlock your Account
+
+To be able to sign transactions, you need to *unlock* your Account in your node:
+
+```
+.account:unlock <validator_address>
```
Comment (ines-toupeira) [inline docs/becoming_validator.md]: there is a dot before "account" that can be removed

File: docs/becoming_validator.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+with at least the validator deposit fee (10 000 NIM). This guide assumes that this amount is present in the validator
+address. To check if that is the case, use:
+
+```
+account:get <validator_address>
+```
+
+
+This guide assumes that this condition is met and that the client that this RPC commands will be run for is already
+configured to validate (check [this guide](becoming_validator#22-configure) to setup you client).
```
Comment (ines-toupeira) [inline docs/becoming_validator.md]: this is a broken link

Comment (jsdanielh) [reply docs/becoming_validator.md]: Fixed

File: docs/becoming_validator.md (hunk 2 above)
Comment (jsdanielh) [reply docs/becoming_validator.md]: Fixed

File: docs/becoming_validator.md (hunk 1 above)
Comment (jsdanielh) [reply docs/becoming_validator.md]: Thanks. Just fixed it.

**Review by viquezclaudio — CHANGES_REQUESTED** [review]
Pending final URL definition

File: docs/becoming_validator.md [hunk 4]
```diff
@@ -67,17 +67,30 @@ validator_address = "NQXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX"
...
 
-Replace the validator address with the one generated in [the previous step](#21-generating-your-validator-address-and-keys) and 
-then uncomment the `signing_key`, the `fee_key` and the `voting_key` fields. Put the respective private keys that were generated
-in [the previous step](#21-generating-your-validator-address-and-keys).
+Replace the validator address and keys generated accordingly.
 
-### 2.3 Sync the Blockchain
+### 2.3 TLS Certificate
+
+It is stronly recommended to use a TLS certificate, because in order for the nimiq-pos wallet to connect to a validator, it needs to establish a secure connection. In order to mantain a healthy decentralization level within the network, it is advisable for the nimiq-pos wallet to connect to as many diverse validators as possible.
```
Comment (jsdanielh) [inline docs/becoming_validator.md]: It is **strongly**...
In order to **maintain**

File: docs/becoming_validator.md [hunk 5]
```diff
@@ -67,17 +67,30 @@ validator_address = "NQXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX"
...
+There are different services where a TLS certificate can be obtained, such as [Let's Encrypt](https://letsencrypt.org/).
+
+Once the certificate is obtained, it can be specified in the Network-TLS section within the config file:
+
+```
+[network.tls]
+private_key = "./my_private_key.pem"
+certificates = "./my_certificate.pem"
+```
+Note: The full chain of certificates necessary to verify the certificate's validity can be specified in the `pem` file. Usually this full chain can be provided by the service that is used to create the TLS certificate.
```
Comment (jsdanielh) [inline docs/becoming_validator.md]: Usually this full chain **is** provided

File: docs/becoming_validator.md [hunk 6]
```diff
@@ -98,22 +111,18 @@ status
...
 
 ```
 account:get <validator_address>
 ```
 
-
-This guide assumes that this condition is met and that the client that this RPC commands will be run for is already
-configured to validate (check [this guide](#22-configure) to setup you client).
-
+Note that validators are selected to produce blocks every epoch (every election block), so it make take some time for your validator to be elected to produce blocks.
```
Comment (jsdanielh) [inline docs/becoming_validator.md]: it **may** take

**Pull request merged**